README.md

```diff
@@ -30,14 +31,27 @@ Historical table of version comparisons between releases of this module and AiiD
 Furthermore, the legacy export versions will never change, i.e., the incremental migration functions need only be thoroughly tested once, and will therefore not be affected by changes to the core of the AiiDA code in any way.
 
 **Q: What happens when the export version is upped?**  
-**A:** A new export archive file will be added to this repo as well as a new test-filled file (to [aiida-core](https://github.com/aiidateam/aiida-core) under `aiida.backends.tests.tools.importexport.migration.`).
+**A:** A new export archive file will be added to this repo as well as a new test-filled file (to [aiida-core](https://github.com/aiidateam/aiida-core) under `tests.tools.importexport.migration.`).
 
 **Q: What if the import system changes in AiiDA core?**  
 **A:** This repo is only for storing the export archives and their creation workflows for different export versions.
 All tests can be found in [aiida-core](https://github.com/aiidateam/aiida-core).
 
 ## Release notes
 
+### 0.9.0 (April 2020)
+
+**AiiDA version**: _1.1.1_
```

Comment:
Shouldn't that be `1.2.0`? It has not yet been released, but that will almost certainly be the version. It doesn't actually exist in 1.1.1

Reply:
I know - but this is the convention I have been using, because it's difficult to pinpoint it otherwise.
Indeed, I have mentioned the _expected_ version, where this will occur, just below (and set it to v1.2.0).

Otherwise, it may be a good idea to update the README and release notes here with hindsight after releases.

Reply:
It should maybe be "Newest AiiDA version released at this point" or something... but that sounds silly :)

Reply:
I see, and this package _has_ to be released before the new `aiida-core` can be released? If it really is this chicken-or-egg situation, we can keep it like this, but maybe it would still be useful to update after the fact and keep for the most recent something like `'unreleased'`.

Reply:
Not a bad idea, for sure.
And yes, this is a female-post-dinosaur-reproductive-issue indeed.

Reply:
I have updated the README and release notes for the current update, and have created issue #12 to address going through previous versions.